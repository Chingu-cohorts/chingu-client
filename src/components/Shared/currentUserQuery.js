import gql from "graphql-tag";

const currentUserQuery = gql`
  query currentUser {
    user {
      id
    }
  }
`;

export default currentUserQuery;
