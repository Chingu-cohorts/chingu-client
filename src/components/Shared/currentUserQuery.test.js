import currentUserQuery from "./currentUserQuery";

it("should be the correct query", () => {
  expect(currentUserQuery).toMatchSnapshot();
});
