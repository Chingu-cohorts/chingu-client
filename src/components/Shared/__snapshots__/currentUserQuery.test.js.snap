// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be the correct query 1`] = `
Object {
  "definitions": Array [
    Object {
      "directives": Array [],
      "kind": "OperationDefinition",
      "name": Object {
        "kind": "Name",
        "value": "currentUser",
      },
      "operation": "query",
      "selectionSet": Object {
        "kind": "SelectionSet",
        "selections": Array [
          Object {
            "alias": null,
            "arguments": Array [],
            "directives": Array [],
            "kind": "Field",
            "name": Object {
              "kind": "Name",
              "value": "user",
            },
            "selectionSet": Object {
              "kind": "SelectionSet",
              "selections": Array [
                Object {
                  "alias": null,
                  "arguments": Array [],
                  "directives": Array [],
                  "kind": "Field",
                  "name": Object {
                    "kind": "Name",
                    "value": "id",
                  },
                  "selectionSet": null,
                },
              ],
            },
          },
        ],
      },
      "variableDefinitions": Array [],
    },
  ],
  "kind": "Document",
  "loc": Object {
    "end": 53,
    "start": 0,
  },
}
`;
